book/src/products/mosaic/fragmentation-in-layer-2.md

```diff
@@ -0,0 +1,17 @@
+# Fragmentation in DeFi's Layer 2
+
+---
+
+While the Ethereum protocol’s revolutionary smart contract functionality is what makes DeFi possible, the ETH transaction speed for DApps is [notoriously slow](https://www.coindesk.com/learn/ethereum-101/will-ethereum-scale), and it stands in the way of further DeFi development and mass adoption. Just as a point of reference, Ethereum’s transaction throughput is about [1,500 times](https://hackernoon.com/the-blockchain-scalability-problem-the-race-for-visa-like-transaction-speed-5cce48f9d44) slower than that of Visa and about [250,000 times](https://www.barrons.com/articles/alibaba-records-25-3-billion-in-singles-day-sales-1510538618) slower than Alipay’s transaction throughput. 
+
+Several blockchain scaling solutions have been proposed to solve this scalability problem, including [State Channels](https://l4.ventures/papers/statechannels.pdf), [Sharding](https://www.bubifans.com/ueditor/php/upload/file/20181015/1539597837236127.pdf), [Plasma](https://plasma.io/plasma.pdf), [OVM](https://optimism.io/), and [ZKRollups](https://github.com/PlasmNetwork/ZKRollups). These scaling solutions can be categorized as Layer 2 scaling solutions.
```

Comment:
The link `Sharding` loads infinitely for me, can someone check?

Reply:
Not sure what to do about this one. this is the link that was in the original doc. Perhaps I should just remove the link?

Reply:
I've decided to remove the link